### PR TITLE
Refactor formplayer breadcrumbs

### PR DESCRIPTION
**Pull request by benrudolph**

### CODE REVIEW

File: src/main/java/beans/exceptions/BaseExceptionResponseBean.java [hunk 1]
```diff
@@ -10,6 +10,8 @@
     protected String status;
     protected String url;
     protected String type;
+    // For backwards compatibility with old CloudCare, we need to add the message property
```
Comment (wpride) [inline src/main/java/beans/exceptions/BaseExceptionResponseBean.java]: Do we care about this now that we disabled the "Use formplayer server" flag?

Comment (benrudolph) [inline src/main/java/beans/exceptions/BaseExceptionResponseBean.java]: unfortunately (or fortunately) both old and new cloudcare utilize the same form entry javascript code and that javascript code expects `message`. we could just change the JS to look for `.exception` as well

Comment (benrudolph) [inline src/main/java/beans/exceptions/BaseExceptionResponseBean.java]: actually changing the javascript code is easier

**Comment by benrudolph** [comment]
updated, can be merged before the other PR

**Pull request merged**